### PR TITLE
PLAT-9309: Upgrade to openapi 3

### PR DESCRIPTION
**Pull request by math-a3k**
- Use drf-spectacular instead of drf-yasg
- Update Pipfile and Pipfile.lock

### CODE REVIEW

File: {{cookiecutter.project_slug}}/Pipfile [hunk 1]
```diff
@@ -32,3 +31,4 @@ google-cloud-secret-manager = "==2.12.6"
 google-auth = "==2.12.0"
 google-cloud-storage = "==2.5.0"
 pipenv = "*"
+drf-spectacular = "*"
```
Comment (danielsousaio) [inline {{cookiecutter.project_slug}}/Pipfile:34]: Would it be better to lock this to some semver range? Minor range should do the trick.

Comment (math-a3k) [reply {{cookiecutter.project_slug}}/Pipfile]: Done!

**Comment by math-a3k** [comment]
I have, give me some minutes for testing

**Review by danielsousaio — APPROVED** [review]
Looks good

**Pull request merged**